static/sass/styles.scss

```diff
@@ -1300,7 +1300,7 @@ $color-link-dark: #69c !default;
 .p-image__background--3-2 {
   @extend .u-aspect-ratio--3-2;
 
-  background-color: rgba(0, 0, 0, 0.3);
+  background-color: rgba(0, 0, 0, 0.03);
```

Comment:
Just use the correct class = we have something in styles.scs called p-image-wrapper_2-3 or something like that - please don't hardcode.